font/ttf.ksy

```diff
@@ -242,6 +242,128 @@ types:
               - id: repeat_value
                 type: u1
                 if: repeat
+  gvar:
+    doc: >
+      gvar - Glyph Variations Table
+      This table provides all of the variation data that describe how TrueType glyph outlines in a 'glyf' table change across the font’s variation space.
+    seq:
+      - id: major_version
+        type: u2
+      - id: minor_version
+        type: u2
+      - id: axis_count
+        type: u2
+      - id: shared_tuple_count
+        type: u2
+      - id: shared_tuples_offset
+        type: u4
+      - id: glyph_count
+        type: u2
+      - id: reserved_flags
+        type: b15
+      - id: long_offsets
```

Comment:
I usually put them into a separaty type of even count of bytes, if it is possible. This makes it easier to calculate offsets in mind and  also makes it easier to edit the type. And don't forget about endianness, it's not handled automatically for now, so if someone wanted `be` it'd be easier to handle that case if you moved the flags into a separate subtype.